AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1683,6 +1683,18 @@ - (void)recursivelyClearFetchedData
   [self clearFetchedData];
 }
 
+- (ASInterfaceState)interfaceState
+{
+  return _interfaceState;
+}
+
+- (void)setInterfaceState:(ASInterfaceState)interfaceState
+{
+  if (interfaceState != _interfaceState) {
```

Comment:
Might be helpful to introduce a callback method: `-[ASDisplayNode interfaceStateDidChange:(ASInterfaceState)previousState]` for subclassing.